crates/copilot/src/copilot_chat.rs

```diff
@@ -31,6 +31,8 @@ pub enum Role {
 #[derive(Clone, Debug, Default, Serialize, Deserialize, PartialEq, EnumIter)]
 pub enum Model {
     #[default]
+    #[serde(alias = "gpt-4o", rename = "gpt-4o-2024-05-13")]
```

Comment:
Interestingly, the GitHub API requires you to specify the model precisely here, despite the other models accepting the more generic name. Not sure why 🤷